document-store/src/main/java/org/hypertrace/core/documentstore/expression/operators/RelationalOperator.java

```diff
@@ -1,16 +1,27 @@
 package org.hypertrace.core.documentstore.expression.operators;
 
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
 public enum RelationalOperator {
-  EQ,
-  NEQ,
-  GT,
-  LT,
-  GTE,
-  LTE,
-  IN,
-  CONTAINS,
-  EXISTS,
-  NOT_EXISTS,
-  LIKE,
-  NOT_IN,
+  EQ("="),
+  NEQ("!="),
+  GT(">"),
+  LT("<"),
+  GTE(">="),
+  LTE("<="),
+  IN("IN"),
+  CONTAINS("CONTAINS"),
+  EXISTS("EXISTS"),
+  NOT_EXISTS("NOT EXISTS"),
+  LIKE("~"),
+  NOT_IN("NOT IN"),
```

Comment:
```suggestion
  NOT_IN("NOT IN");
```

Reply:
nit: if you do this remove the below the line.

Reply:
Prefer to keep semicolon in new line for allowing operator extendibility without affecting the git blame of current line.